DCode.Web/Scripts/App_JS/Controllers/Requestor.js

```diff
@@ -558,9 +571,9 @@
                 isValid = false;
             }
             else {
-                var regex = /^[a-zA-Z]{3,}[0-9]{5,}[-]{1,}[0-9]{2,}[-]{1,}[0-9]{2,}[-]{1,}[0-9]{2,}[-]{1,}[0-9]{4,}$/;
+                var regex = /^[a-zA-Z]{3,}[0-9]{5,}[-]{1,}[0-9]{2,}[-]{1,}[0-9]{2,}[-][0-9]{2,}[-]{1,}[0-9]{4,}$/;
                 var val = $("#txtWBSCode").val().toLocaleLowerCase();

```

Comment:
RegEx Code got changed. Please fix this.

Reply:
@ritwiksen

Reply:
The updated regex has been pushed by Ritwik.